internal/backoff/backoff.go

```diff
@@ -25,3 +25,12 @@ import "time"
 // Strategy defines a function that will return a
 // backoff time given an attempt number.
 type Strategy func(attempts uint) time.Duration
+
+// FixedBackoff is a constant backoff time that will be used
+// for every backoff time (regardless of the attempt number).
+type FixedBackoff time.Duration
```

Comment:
I **really** dislike this kind of typedeffing, it leads to casting eveywhere.

Why not:

```go
func FixedBackoff(duration time.Dutation) Strategy {
  return func(uint) time.Duration {
    return duration
  }
}
```